OutOfSchool/OutOfSchool.WebApi/Services/ProviderServices/ProviderService.cs

```diff
@@ -421,41 +394,43 @@ public async Task<bool> IsBlocked(Guid providerId)
         return (await providerRepository.GetById(providerId).ConfigureAwait(false)).IsBlocked;
     }
 
-    public async Task<ProviderLicenseStatusDto> UpdateLicenseStatus(ProviderLicenseStatusDto dto, string userId)
+    public async Task SendNotification(Provider provider, NotificationAction notificationAction, bool addStatusData, bool addLicenseStatusData)
```

Comment:
(Performance, readability) You could remove async/await pair in this method, as only awaitable operation is last one.